app/src/main/java/com/rentminder/MainActivity.kt

```diff
@@ -278,7 +284,8 @@ fun EditBillAmounts() {
                         .height(55.dp),
                     textStyle = TextStyle.Default.copy(fontSize = 18.sp))
                 SaveRemindButton(
-                    enabled = otherBill.isNotEmpty(), inputEdited = otherBillEdited
+                    enabled = otherBill.isNotEmpty(),
+                    inputEdited = otherBillEdited
```

Comment:
Formatting can, and should, be performed automatically, and consistently, by the IDE.  Formatting code is really programmer's preference anyway, and doesn't necessarily decrease technical debt.